acceptance/testdata/builder/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:bionic
+
+ARG cnb_uid=1234
+ARG cnb_gid=1000
+ARG cnb_platform_api
+
+ENV CNB_PLATFORM_API=${cnb_platform_api}
```

Comment:
We could probably remove this and just set the `CNB_PLATFORM_API` env var when running the container (using `--env`).

Reply:
Got it

Reply:
fixed